Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -500,6 +506,31 @@ public static Branch getTestInstance(Context context) {
     @TargetApi(Build.VERSION_CODES.ICE_CREAM_SANDWICH)
     public static Branch getAutoInstance(Context context) {
         isAutoSessionMode_ = true;
+        customReferrableSettings_ = CUSTOM_REFERRABLE_SETTINGS.USE_DEFAULT;
+        boolean isDebug = BranchUtil.isTestModeEnabled(context);
+        getBranchInstance(context, !isDebug);
```

Comment:
this might be a little nitpicky, but perhaps rename isDebug to isLive, and have the `!` on the assign line, vs inside the method call line? it's a bit more clear that way, but again just a slight nitpick